bears/js/JSHintBear.py

```diff
@@ -16,19 +31,377 @@ class JSHintBear(LocalBear, Lint):
+                                                about invalid ``typeof``
+                                                operator values.
+        :param es3:                             This option tells JSHint that
+                                                your code needs to adhere to
+                                                ECMAScript 3 specification.
+        :param force_filter_forin:              This option requires all
+                                                ``for in`` loops to filter
+                                                object's items.
+        :param iterator:                        This option suppresses warnings
+                                                about the ``__iterator__``
+                                                property.
+        :param allow_funcscope:                 This option suppresses warnings
+                                                about declaring variables inside
+                                                of control structures while
+                                                accessing them later from
+                                                outside.
+        :param prohibit_arg:                    This option prohibits the use of
+                                                ``arguments.caller`` and
+                                                ``arguments.callee``.
+        :param prohibit_comma:                  This option prohibits the use of
+                                                the comma operator.
+        :param prohibit_non_breaking_whitespace: This option warns about "non-
+                                                breaking whitespace characters"
+        :param prohibit_new:                    This option prohibits the use of
+                                                constructor functions for
+                                                side-effects.
+        :param prohibit_undefined:              This option prohibits the use of
+                                                explicitly undeclared variables.
+        :param prohibit_groups:                 This option prohibits the use of
+                                                the grouping operator when it is
+                                                not strictly required.
+        :param prohibit_variable_statements:    This option forbids the use of
+                                                VariableStatments.
+        :param allow_missing_semicol:           This option suppresses warnings
+                                                about missing semicolons.
+        :param allow_debugger:                  This option suppresses warnings
+                                                about the ``debugger``
+                                                statements.
+        :param allow_assignmentcomparisions:    This option suppresses warnings
```

Comment:
I think you should introduce another underscore --> `allow_assignment_comparisions`